sdk-wallet/validator_wallet_provider.md

```diff
@@ -21,4 +24,8 @@ class ValidatorWalletProvider:
     // Can throw:
     // - ErrInvalidPublicKey
```

Comment:
`ErrInvalidPublicKey` -> `ErrInvalidPublicKeyBytes`, maybe? Since the bytes are invalid

Reply:
Fixed.